zaza/openstack/charm_tests/octavia/tests.py

```diff
@@ -99,11 +100,77 @@ def revoke_role_current_user(keystone_client, keystone_session, role_name,
 class CharmOperationTest(test_utils.OpenStackBaseTest):
     """Charm operation tests."""
 
+    application_name = "octavia"
+    states = {"octavia": {
```

Comment:
You don't need this, see other comment, thanks!

Reply:
done